composer.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Hi @gnovaro ,
composer file available in the following location.. 
symfony/lib/composer.json
how to update dependencies described here 
https://github.com/orangehrm/orangehrm/wiki/Development

Reply:
Excuse me is and strange folder structure. I expected something like public folder. composer and vendor in the root.

Reply:
That's obvious .. This is 15 Y of old code..  deprecating old stuff more carefully,, 
most importantly, This composer changes not address the project dependencies.